components/omega/lint-omega.txt

```diff
@@ -0,0 +1,11 @@
+# This file may be used to create an environment for linting Omega with
+# pre-commit using:
+# $ mamba create -n omega_dev --file <this file>
```

Comment:
Maybe this file should be `dev-conda.txt` or something like that.  In which case it maybe does belong in `components/omega` directly.

Reply:
Yeah, I think any of these belong in the components/omega top directory. That's where the Cmake files will sit too.